### ISSUE TITLE
Panel: Screen reader does not convey any information after activating 'Save' button under 'Panel - Small, right-aligned, fixed, with footer' pane. 

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10777]
[Accessibility][Screen Reader : Panel Control] : Screen reader does not convey any information after activating the 'Save' button.

**User Impact:**
Screen reader users will be impacted if screen reader will not inform to users that task has been completed or not after activating the 'Save'  button.

**Usage:**
1.Panel - Small Panel, Anchored Right, Fixed Width
2.Panel - Footer

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 18999.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/panel
Edge Browser Version:
Microsoft Edge: 44.18999.1.0
Microsoft EdgeHTML: 18.18999
Chrome Browser Version:
Chrome Version: 76.0.3809.132

**Repro Steps:**
1. Open https://developer.microsoft.com/en-us/fabric#/controls/web/panel in Edge and turn on Narrator(Ctrl+Win+Enter).
2. Navigate to ‘Surface' button by using Tab key and activate it.
3. Navigate to 'Panel' link by using Down arrow key and activate it.
4. 'Panel' pane will be appeared.
5. Navigate to 'Open Panel' under usages (Panel - Small Panel, Anchored Right, Fixed Width) by using Tab key and activate it.
6. Panel - Small Panel, Anchored Right, Fixed Width pane will be appeared.
7. Select Option A by using down arrow key.
8. Navigate to Save button by using Tab key.
9. Activate the 'Save' button and verify the issue.

**Actual Result:** 
After activating 'Save' button, screen reader does not convey any information, whether it save successfully or not.

**Expected Result:**
Screen reader should provide the information whether task completed or not. Screen reader should announce 'Saved successfully'.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=9f09d896-88c4-43dc-8bd1-8f74733d86ca

**Attachment:**
[Screen reader does not convey any information.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3711552/Screen.reader.does.not.convey.any.information.zip)
![Screen reader does not convey any information after activating 'Save' button under 'Panel - Small, right-aligned, fixed, with footer' pane](https://user-images.githubusercontent.com/55537167/66556831-87df5080-eb6e-11e9-92fa-0c3809dc37cb.png)

**Comment by ecraig12345** [comment]
I'm not convinced this is something we need to fix--any hypothetical "saving" action is tangential to the point of the example. The example's actual point is just to demonstrate how to code a panel with a footer, and what it looks like.
